client/components/TeamOutline.js

```diff
@@ -14,7 +14,7 @@ const TeamOutline= ({page}) => {
                   {/*Staff Coordinator*/}
                   <div className="container staff-lead">
                     <div className="row">
-                      <h2 className="team-row-heads ">STAFF COORDINATOR</h2>
+                      <h2 className="team-row-heads ">{team.section_1}</h2>
```

Comment:
You could've used array.map() if possible, instead of hard-coding for 18 sections

Reply:
Indeed, a brief discussion might be required before moving ahead.